Resources/Prototypes/Entities/Effects/puddle.yml

```
@@ -21,7 +21,7 @@
   - type: SolutionContainerManager
     solutions:
       puddle:
-        maxVol: 1000
+        maxVol: 1200 # Carpmosia-edit - extendedReagentStorage
```

Setting the maxVol accross all these puddles when it's set on the parent is cursed. If it's inherited it shouldn't be there but since these are upstream lines the maxVols should be commented out.
```suggestion
        #        maxVol: 1200 # Carpmosia-edit - extendedReagentStorage
```